packages/plugin-core/package.json

```diff
@@ -163,17 +159,23 @@
         "title": "Dendron: Rename Header",
         "desc": "Rename a header, and update all links to it."
       },
+      {
+        "command": "dendron.convertLink",
+        "title": "Dendron: Convert Link",
+        "desc": "Convert link candidate into backlink"
+      },
       {
         "command": "dendron.lookupNote",
-        "title": "Dendron: Lookup Note",
+        "title": "Dendron: Lookup",
```

Comment:
this is a mistake on our part. can you update the `constants` file to update the title here?